src/main/java/com/iota/iri/conf/MilestoneConfig.java

```diff
@@ -15,9 +17,28 @@
      */
     boolean isDontValidateTestnetMilestoneSig();
 
+    /**
+     * @return {@value Descriptions#NUMBER_OF_KEYS_IN_A_MILESTONE}
+     */
+    int getNumberOfKeysInMilestone();
```

Comment:
Maybe add default value to the description like in https://github.com/iotaledger/iri/pull/1159?